docs/installing/index.md

```diff
@@ -6,61 +6,96 @@ title: Installing
+means on your local machine.
+
+So the best way to get started is to 
+[install locally using Vagrant]({{ page.baseurl }}/docs/installing/vagrant/)
+— that's easiest because it takes care of local dependencies for you.
+In order to customise your installation, you'll need to make your own theme
+and, eventually, that theme will go into its own
+<a href="{{ page.baseurl}}/docs/glossary/#git" class="glossary__link">git
+repo</a>. Only when you've got this far can you move onto a
+<a href="{{ page.baseurl}}/docs/glossary/#staging" class="glossary__link">staging site</a>.
+Here you can test the code in an identical environment to your production site.
+The very last step is to go live.
+
+<img src="{{page.baseurl}}/assets/img/alaveteli-install-path.svg" />
+
+Depending on the resources you have available, it might be that your staging server becomes your production server.
+
+## How to install the Alaveteli code
+
+* [Install into a Vagrant virtual development environment]({{ page.baseurl }}/docs/installing/vagrant/)
+  -- the easiest way to get a development version up and running
+
+<div class="attention-box info">
+    Although we recommend Vagrant for development, there are of course other ways
+    to install Alaveteli. Vagrant is never suitable for production (but remember
+    that you won't need a production site until you've done a development
+    deployment). We've made an Amazon Machine Image (AMI) so you can quickly
+    deploy on Amazon EC2. If you prefer to use your own server, there's an
+    installation script which does most of the work for you, or you can follow
+    the manual installation instructions.
+</div>
+<div class="attention-box helpful-hint">
+    <strong>
+      If you're not sure which one you want,
+      <a href="{{ page.baseurl }}/docs/installing/vagrant/">install using
+      Vagrant</a> first!
+    </strong>
+</div>
 
-If you're in doubt, you're probably running a development site. Get it up and
-running, play with it, customise it, and -- later -- you can install it as a
-production server.
-
-## Deployment
-
-If you're running a production server, we **strongly recommend** you
-use the Capistrano [deployment mechanism]({{ page.baseurl }}/docs/installing/deploy/)
-that's included with Alaveteli. Set this up and you never have to edit files on
-those servers, because Capistrano takes care of that for you.
-
-## Installing the core code
-
-* [Install into a Vagrant virtual development environment]({{ page.baseurl }}/docs/installing/vagrant/) -- a good choice for development, and playing around with the site.
 * [Install on Amazon EC2]({{ page.baseurl }}/docs/installing/ami/) using our AMI
 * [Use the installation script]({{ page.baseurl }}/docs/installing/script/) which does the full installation on your own server
 * [Manual installation]({{ page.baseurl }}/docs/installing/manual_install/) -- step-by-step instructions
 
+<!--
 If you're setting up a development server on MacOS X, we've also got
```

Comment:
I think you're right to remove this text - keeping it up to date is a nightmare (applies equally to the text and the MacOS setup once you've got it). Using Vagrant is easier and a better stand-in for a live environment